api/docs.go

```diff
@@ -20,6 +20,49 @@ const docTemplate = `{
     "host": "{{.Host}}",
     "basePath": "{{.BasePath}}",
     "paths": {
+        "/popular_repositories/": {
+            "get": {
+                "description": "get public repositories",
```

Comment:
```suggestion
                "description": "Get public repositories",
```
Most other descriptions start with a capital letter